threadsd/main.go

```diff
@@ -128,6 +139,9 @@ func main() {
 	} else {
 		opts = append(opts, common.WithNetBadgerPersistence(*repo))
 	}
+	if announceAddrStr != nil {
```

Comment:
should be `if announceAddr != nil {` ?

Reply:
Thank you - I missed it. GoLand probably hinted the var and I accepted it without paying attention.
I fixed the error!